### ISSUE TITLE
Crash after clicking URL in Welcome to reddit message

### ISSUE THREAD

**Issue by karlding** [ccrama/Slide#364]
Clicking on the hyperlink in the **Welcome to reddit** message that asks you to **Type in an interest** causes a crash. In a browser, the URL should bring you to a page where you can subscribe to new subreddits (not sure how you'd want to handle that in an app&mdash;I could see it getting ugly fast). I've included a stack trace, since the GitHub reporter didn't show up.

This is the relevant URL: ``https://www.reddit.com/subreddits?ref=welcomepm``

```
--------- beginning of crash

11-05 00:13:14.549 28314 28344 E AndroidRuntime: FATAL EXCEPTION: AsyncTask #2

11-05 00:13:14.549 28314 28344 E AndroidRuntime: Process: me.ccrama.redditslide, PID: 28314

11-05 00:13:14.549 28314 28344 E AndroidRuntime: java.lang.RuntimeException: An error occurred while executing doInBackground()

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at android.os.AsyncTask$3.done(AsyncTask.java:309)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at java.util.concurrent.FutureTask.setException(FutureTask.java:223)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at java.util.concurrent.FutureTask.run(FutureTask.java:242)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:234)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:818)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: Caused by: net.dean.jraw.http.NetworkException: Request returned non-successful status code: 404 Not Found

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at net.dean.jraw.http.RestClient.execute(RestClient.java:132)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at net.dean.jraw.RedditClient.execute(RedditClient.java:147)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at net.dean.jraw.RedditClient.execute(RedditClient.java:141)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at net.dean.jraw.RedditClient.getSubmission(RedditClient.java:344)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at net.dean.jraw.RedditClient.getSubmission(RedditClient.java:322)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at me.ccrama.redditslide.Activities.CommentsScreenSingle$AsyncGetSubredditName.doInBackground(CommentsScreenSingle.java:48)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at me.ccrama.redditslide.Activities.CommentsScreenSingle$AsyncGetSubredditName.doInBackground(CommentsScreenSingle.java:29)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at android.os.AsyncTask$2.call(AsyncTask.java:295)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)

11-05 00:13:14.549 28314 28344 E AndroidRuntime: 	... 4 more

11-05 00:13:14.556   799  3521 W ActivityManager:   Force finishing activity me.ccrama.redditslide/.Activities.Crash

11-05 00:13:14.563   799  3521 W ActivityManager:   Force finishing activity me.ccrama.redditslide/.Activities.CommentsScreenSingle
```

**Comment by ccrama** [comment]
Yeah I really don't know how I could handle this in app at all. I don't know how I could fix this except for just opening the app by itself

**Comment by Alexendoo** [comment]
I would suggest either having it just open the app at least for now so it doesn't cause a crash at least, you could have it open the 'go to subreddit' field for extra fanciness since that's (in a sense) what it does.

**Comment by Alexendoo** [comment]
Will tag for release, to handle the crash at least

**Comment by ccrama** [comment]
Fixed b356a5b
